vmdb/app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -53,12 +52,16 @@ def textual_portal_ip
     {:label => "Portal IP", :value => @record.portal_ip}
   end
 
-  def textual_protocol
-    {:label => "Protocol", :value => @record.protocol}
-  end
 
-  def textual_container_port
-    {:label => "Container Port", :value => @record.container_port}
+  def textual_port_config(single_conf)
+    name = single_conf.name
+
+    name = "Unnamed" if name.nil? || name.empty?
+
+    {
+      :label => name,
+      :value => "#{single_conf.protocol} port:#{single_conf.port} to port:#{single_conf.target_port}"
```

Comment:
@zeari , maybe it will be more clear if the second part will be "to target port X" instead of "to port"
it will also resonate better with the original field name.